apps/extension/src/final/extension-popup/extension-popup.tsx

```diff
@@ -1,55 +1,78 @@
 import { Route, Routes } from 'react-router';
+import { useState } from 'react';
 
 import { Closable } from 'shared/ui';
 import { POPUP_ROUTE, useExtensionPopup } from 'shared/extension';
+import { SubscriptionsManagement } from 'application/trading-copilot';
 
-import {
-  TopBar,
-  Footer,
-  PopupContentLayout,
-  SettingsLayout,
-} from './components';
 import {
   GovernanceSettings,
   MainSettings,
   OtherSettings,
   Products,
 } from './views';
+import {
+  TopBar,
+  Footer,
+  PopupContentLayout,
+  SettingsLayout, 
+  TradingCopilotLayout,
+} from './components';
+import { TradingCopilotToasts } from './components/trading-copilot-toasts';
```

Comment:
we can add an export of `TradingCopilotToasts` and `TradingCopilotDialogs` in the `extension-popup/components/index.ts` and import both from `./components`